common/lib/xmodule/xmodule/graders.py

```diff
@@ -151,7 +151,7 @@ def invalid_args(func, argdict):
     Given a function and a dictionary of arguments, returns a set of arguments
     from argdict that aren't accepted by func
     """
-    args, _, keywords, _ = inspect.getargspec(func)
+    args, _, keywords, _, _, _, _ = inspect.getfullargspec(func)
```

Comment:
Why not using `inspect.signature`?

Reply:
As per documentation https://docs.python.org/3/library/inspect.html#inspect.getargspec, it is recommended that we use `inspect.getfullargspec`, `inspect.signature` returns a `Signature` object and we don't need that here

> Deprecated since version 3.0: Use getfullargspec() for an updated API that is usually a drop-in replacement, but also correctly handles function annotations and keyword-only parameters.